bin/package-and-publish-helm-chart.py

```diff
@@ -0,0 +1,148 @@
+import os
+import re
+import requests
+import subprocess
+import sys
+
+from shutil import copyfile
+
+
+def main():
+  args = sys.argv[1:]
+  chart_path = args[0]
+  working_directory = args[1]
+  build_number = args[2]
+  is_prerelease = args[3]
+  application_version = args[4]
+
+  chart_filename = chart_path + '/Chart.yaml'
+
+  print(f'build_number = {build_number}')
+  print(f'is_prerelease = {is_prerelease}')
+  print(f'application_version = {application_version}')
+
+  username = os.getenv('ARTIFACTORY_USER')
+  token = os.getenv('ARTIFACTORY_TOKEN')
+
+  # Parse chart name from the Chart.yaml
+  chart_name = None
+  with open(chart_filename) as file:
+    for line in file:
+      if line.startswith('name:'):
+        chart_name = line.split(':')[1].strip()
+        print(f'Chart name: {chart_name}')
+        break
+
+  # Parse version from the Chart.yaml
+  chart_version = '0.0.0'
+  with open(chart_filename) as file:
+    for line in file:
+      if line.startswith('version:'):
+        chart_version = line.split(':')[1].strip()
+        print(f'Chart version: {chart_version}')
+        break
+
+  version_elements = chart_version.split('.')
+  version_major = int(version_elements[0])
+  version_minor = int(version_elements[1])
+  version_patch = int(version_elements[2])
```

Comment:
```suggestion
  version_major, version_minor, version_patch = map(int, chart_version.split('.'))
```
This logic is repeated below, so you might make it another function, maybe?

Reply:
oh, except that it won't work if chart_version is something like 1.2.3.dev4  .. in that case you might add [:3] in there to limit the split() to only 3 elements